package.json

```diff
@@ -35,34 +35,17 @@
 		"@babel/preset-env": "7.4.2",
 		"@babel/preset-react": "7.0.0",
 		"@babel/runtime": "7.4.2",
-		"@wordpress/a11y": "2.0.2",
-		"@wordpress/api-fetch": "2.2.8",
-		"@wordpress/babel-plugin-import-jsx-pragma": "1.1.3",
-		"@wordpress/blob": "2.1.0",
 		"@wordpress/block-library": "2.2.16",
 		"@wordpress/blocks": "6.0.7",
 		"@wordpress/browserslist-config": "2.3.0",
 		"@wordpress/components": "7.0.8",
 		"@wordpress/compose": "3.0.1",
-		"@wordpress/core-data": "2.0.17",
-		"@wordpress/data": "4.2.1",
 		"@wordpress/date": "3.0.1",
-		"@wordpress/deprecated": "2.0.5",
-		"@wordpress/dom": "2.0.8",
 		"@wordpress/edit-post": "3.1.11",
-		"@wordpress/editor": "9.0.11",
```

Comment:
This is still imported in some `/gutenberg/extensions`:
- editor-notes
- todo
- trial
- trials

Reply:
Good catch! I was planning to land this after https://github.com/Automattic/wp-calypso/pull/31954 which removes those usages.